packages/waas/src/payloads/signers/index.ts

```diff
@@ -0,0 +1,28 @@
+import { newSECP256K1RandomSigner, newSECP256K1Signer } from "./secp256k1";
+import { newSECP256R1Signer, newSECP256R1RandomSigner } from "./secp256r1";
+
+export type PayloadSigner = {
+  privateKey: string
+
+  verifier(): Promise<string>
```

Comment:
I propose, rename `verifier(): Promise<string>` to just `keyType(): string` and have it return "secp256k1" or "secp256r1" or instead of "keyType()" name, use "alg()" aka algorithm, similar to what jwt calls it

Reply:
Yes, the SessionSigner interface should look along these lines:
```ts
export type SessionSigner = {
    keyType(): string // the alg
    publicKey(): string // the public key for verification
    sign(...): string // the signing function
    verify(): string // the verification function
}
```